### PR TITLE
Close query runners properly in various tests

### PR DESCRIPTION
**Pull request by nezihyigitbasi**

### CODE REVIEW

File: presto-tests/src/test/java/com/facebook/presto/tests/TestTpchDistributedStats.java [hunk 1]
```diff
@@ -44,20 +46,29 @@
...
+    public void init()
             throws Exception
     {
         runner = createQueryRunnerWithoutCatalogs(emptyMap(), emptyMap());
         runner.createCatalog("tpch", "tpch", ImmutableMap.of(
                 "tpch.column-naming", ColumnNaming.STANDARD.name()));
     }
 
+    @AfterClass(alwaysRun = true)
+    public void shutdown()
```
Comment (findepi) [inline presto-tests/src/test/java/com/facebook/presto/tests/TestTpchDistributedStats.java]: nit: these are usually called `setUp` and `tearDown` in tests, i believe

Comment (nezihyigitbasi) [reply presto-tests/src/test/java/com/facebook/presto/tests/TestTpchDistributedStats.java]: I see a mix of those, but I guess setup/teardown are more common. thanks!

**Comment by nezihyigitbasi** [comment]
@haozhun the tests look good with these fixes.

File: presto-tests/src/test/java/com/facebook/presto/execution/resourceGroups/db/TestEnvironments.java [hunk 2]
```diff
@@ -31,53 +29,37 @@
...
 
+@Test(singleThreaded = true)
 public class TestEnvironments
 {
     private static final String LONG_LASTING_QUERY = "SELECT COUNT(*) FROM lineitem";
-    private DistributedQueryRunner runner;
 
-    @BeforeGroups(value = TEST_ENVIRONMENT)
-    public void setUpRunnerForEnvironment1()
+    @Test(timeOut = 240_000, groups = TEST_ENVIRONMENT)
```
Comment (electrum) [inline presto-tests/src/test/java/com/facebook/presto/execution/resourceGroups/db/TestEnvironments.java]: Remove the groups. These are for conditionally running tests and aren't used here at all.

**Comment by electrum** [comment]
Otherwise looks good

File: presto-tests/src/test/java/com/facebook/presto/execution/resourceGroups/db/TestEnvironments.java (hunk 2 above)
Comment (nezihyigitbasi) [reply presto-tests/src/test/java/com/facebook/presto/execution/resourceGroups/db/TestEnvironments.java]: yep, this is leftover.

**Pull request merged**